### PR TITLE
Set up TTL on the `label_change` table

### PR DESCRIPTION
**Pull request by quantranhong1999**

### CODE REVIEW

File: tmail-backend/jmap/extensions-cassandra/src/main/scala/com/linagora/tmail/james/jmap/label/CassandraLabelChangesConfiguration.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import com.google.common.base.Preconditions
+import org.apache.commons.configuration2.Configuration
+import org.apache.james.util.DurationParser
+
+object CassandraLabelChangesConfiguration {
+  private val DEFAULT_TTL: Duration = Duration.ofDays(60)
+  val DEFAULT: CassandraLabelChangesConfiguration = CassandraLabelChangesConfiguration(DEFAULT_TTL)
+
+  def from(configuration: Configuration): CassandraLabelChangesConfiguration = {
+    val labelChangeTtl: Duration = Option(configuration.getString("label.change.ttl", null))
```
Comment (chibenwa) [inline tmail-backend/jmap/extensions-cassandra/src/main/scala/com/linagora/tmail/james/jmap/label/CassandraLabelChangesConfiguration.scala:15]: Where's the doc for this conf?

Comment (quantranhong1999) [reply tmail-backend/jmap/extensions-cassandra/src/main/scala/com/linagora/tmail/james/jmap/label/CassandraLabelChangesConfiguration.scala:15]: It seems TMail does not have a doc entry for additional configuration yet. I would add a docs for those.

File: upgrade-instructions.md [hunk 2]
```diff
@@ -12,6 +12,27 @@ software documentation. Do not follow this guide blindly!
...
+
+From now on, we set a default 60 days (configurable) Cassandra time-to-live on the `label_change` table
+to reduce considered outdated changes' storage usage.
+
+If one want to keep the old behavior which is not using Cassandra time-to-live on those JMAP change tables, please set the
+following configurations in `cassandra.properties`:
+
+```
+label.change.ttl=0
+```
```
Comment (Arsnael) [inline upgrade-instructions.md:32]: You don't precise that the value is in seconds. And should it be seconds or days btw?

Comment (quantranhong1999) [reply upgrade-instructions.md:32]: It defaults to `second`. That is good enough for OPS IMO.

Comment (Arsnael) [reply upgrade-instructions.md:32]: Maybe with the configuration doc? Or some examples? It's not trivial that if you put just a number it's in seconds and that you can add the unit of duration as well alongside it for example :)

**Review by Arsnael — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Pull request merged**